### DISCUSSION TITLE
Minio Storage

### DISCUSSION THREAD

**Discussion by RainbowZephyr** [micronaut-projects/micronaut-core#10375] [Q&A] [answered]
Hello,
Does anyone have an idea on how to integrate Minio with Micronaut? It should be S3 compliant but I having a bit of a hard time using AWS S3 configuration provided by the guide for Minio.

Thanks in advance.

**Comment by sdelamo** [comment, marked as answer]
we use it to test the eclipse store integration https://github.com/micronaut-projects/micronaut-eclipsestore/blob/master/gradle/libs.versions.toml#L38 Check that repository and you will find tests using it.
